requirements.txt

```diff
@@ -47,4 +47,5 @@ google-cloud-bigquery
 google-cloud-bigquery-storage
 xvfbwrapper
 deepdiff
-xmltodict
\ No newline at end of file
+xmltodict
+configobj
```

Comment:
We don't need additional configobj to accomplish this. There is already a builtin config handler module inside node for these use cases, please use that.